static/css/material.less

```diff
@@ -152,11 +152,13 @@
 
 @media (max-width: 767px) {
   .material {
-    .action-list,
-    .action-list-title {
+    .action-list {
       margin-left: -15px;
       margin-right: -15px;
     }
+    .xs-block {
+      display: block !important;
```

Comment:
Are you absolutely sure "!important" is necessary? It's really bad form if you can avoid it.